pkg/auth/tokenparser_test.go

```diff
@@ -321,4 +321,39 @@ func (s *TestTokenParserSuite) TestTokenParser() {
 		require.Equal(s.T(), email0, claims.Email)
 		require.Equal(s.T(), "OriginalSubValue:1234-ABCD", claims.OriginalSub)
 	})
+
+	s.Run("parse valid token with aud claim", func() {
```

Comment:
TBH I'm not 100% sure we need to test this here, as this seems more functionality that belongs to the JWT library. And it should be already tested here https://github.com/golang-jwt/jwt/blob/3c0777d0c9b7acb6c7d1ad1ae39ae39b7c370d8f/parser_test.go#L358-L381 , but I may be wrong.

Reply:
As I mentioned above this code basically tests that our tokenparser uses the right jwt version. If we roll back our changes from go.mod we are introducing in this PR then the new test would fail.

Reply:
Right , using the old version would fail because that was expecting `aud` to be a string. My comment was more related to the fact that having a slice with a single string or multiple string doesn't change that much IMO. The field type is the same, but it's just a minor thing.